tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -79,57 +84,56 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
+		defer cancel()
+		var storageclass *storagev1.StorageClass
+		var pvclaim *v1.PersistentVolumeClaim
+		var svcPVCName string
+		var err error
+		ginkgo.By("Creating Storage Class and PVC")
 		// decide which test setup is available to run
-		if supervisorCluster {
+		if vanillaCluster {
+			ginkgo.By("CNS_TEST: Running for Vanilla setup")
+			scParameters[scParamFsType] = ext4FSType
+			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "")
+		} else if supervisorCluster {
 			ginkgo.By("CNS_TEST: Running for WCP setup")
+			ginkgo.By(fmt.Sprintf("storagePolicyName: %s", storagePolicyName))
 			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyName)
 			scParameters[scParamStoragePolicyID] = profileID
 			// create resource quota
 			createResourceQuota(client, namespace, rqLimit, storagePolicyName)
+			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "", storagePolicyName)
 		} else {
 			ginkgo.By("CNS_TEST: Running for GC setup")
 			scParameters[svStorageClassName] = storagePolicyName
-		}
-		var storageclass *storagev1.StorageClass
-		var pvclaim *v1.PersistentVolumeClaim
-		var svcPVCName string // PVC Name in the Supervisor Cluster
-		var err error
-		ctx, cancel := context.WithCancel(context.Background())
-		defer cancel()
-		// decide which test setup is available to run
-		if guestCluster {
-			ginkgo.By("CNS_TEST: Running for GC setup")
+			createResourceQuota(client, namespace, rqLimit, storagePolicyName)
 			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "")
-		} else {
-			ginkgo.By("CNS_TEST: Running for WCP setup")
-			storageclass, pvclaim, err = createPVCAndStorageClass(client, namespace, nil, scParameters, "", nil, "", false, "", storagePolicyName)
 		}
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 		defer func() {
 			err := client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, *metav1.NewDeleteOptions(0))
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
-		defer func() {
-			err := fpv.DeletePersistentVolumeClaim(client, pvclaim.Name, namespace)
-			gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		}()
 
 		ginkgo.By("Waiting for claim to be in bound phase")
 		pvc, err := fpv.WaitForPVClaimBoundPhase(client, []*v1.PersistentVolumeClaim{pvclaim}, framework.ClaimProvisionTimeout)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(pvc).NotTo(gomega.BeEmpty())
-
 		pv := getPvFromClaim(client, pvclaim.Namespace, pvclaim.Name)
 		volumeID := pv.Spec.CSI.VolumeHandle
 
+		defer func() {
+			err := fpv.DeletePersistentVolumeClaim(client, pvclaim.Name, namespace)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
 		if guestCluster {
 			// svcPVCName refers to PVC Name in the supervisor cluster
-			svcPVCName = volumeID
+			svcPVCName := volumeID
```

Comment:
`svcPVCName` was declared in line 93, this action will create a local variable

Reply:
Removed ":"